pigot:

  # Arm controller configuration.
  group_arm_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - arm_rot_1
      - arm_rot_2
      - arm_rot_3
      - arm_rot_4
      - arm_rot_5
      - arm_rot_6
    #gains:
      #arm_rot_1:   {p: 5000.0, i: 0.0, d: 10}
      #arm_rot_2:   {p: 5000.0, i: 0.0, d: 10}
      #arm_rot_3:   {p: 5000.0, i: 0.0, d: 10}
      #arm_rot_4:   {p: 5000.0, i: 0.0, d: 10}
      #arm_rot_5:   {p: 5000.0, i: 0.0, d: 10}
      #arm_rot_6:   {p: 5000.0, i: 0.0, d: 10}

  group_gripper_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - finger_pri_1
      - finger_pri_2
    #gains:
      #finger_pri_1:  {p: 5000.0, i: 0.0, d: 10}
      #finger_pri_2:  {p: 5000.0, i: 0.0, d: 10}



  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  
  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_rot_1
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_rot_2
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: lf_rot_3
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_rot_1
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_rot_2
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: rf_rot_3
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint7_position_controller:
    type: effort_controllers/JointPositionController
    joint: lb_rot_1
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint8_position_controller:
    type: effort_controllers/JointPositionController
    joint: lb_rot_2
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint9_position_controller:
    type: effort_controllers/JointPositionController
    joint: lb_rot_3
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint10_position_controller:
    type: effort_controllers/JointPositionController
    joint: rb_rot_1
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint11_position_controller:
    type: effort_controllers/JointPositionController
    joint: rb_rot_2
    pid: {p: 5000.0, i: 0.01, d: 100.0}

  joint12_position_controller:
    type: effort_controllers/JointPositionController
    joint: rb_rot_3
    pid: {p: 3000.0, i: 0.01, d: 100.0}

pigot/gazebo_ros_control/pid_gains:
      arm_rot_1:   {p: 5000.0, i: 0.01, d: 100.0}
      arm_rot_2:   {p: 5000.0, i: 0.01, d: 100.0}
      arm_rot_3:   {p: 5000.0, i: 0.01, d: 100.0}
      arm_rot_4:   {p: 5000.0, i: 0.01, d: 100.0}
      arm_rot_5:   {p: 5000.0, i: 0.01, d: 100.0}
      arm_rot_6:   {p: 5000.0, i: 0.01, d: 100.0}
      finger_pri_1:  {p: 5000.0, i: 0.01, d: 100.0}
      finger_pri_2:  {p: 5000.0, i: 0.01, d: 100.0}


